pandas/core/style.py

```diff
@@ -771,3 +850,10 @@ def _highlight_extrema(data, color='yellow', max_=True):
                 extrema = data == data.min().min()
             return pd.DataFrame(np.where(extrema, attr, ''),
                                 index=data.index, columns=data.columns)
+
+
+def _maybe_wrap_formatter(formatter):
+    if not callable(formatter):
+        return lambda x: formatter.format(x)
+    else:
+        return formatter
```

Comment:
doesn't this *have* to be a string?

e.g. ``df.style.format(5)`` is a failure, yes? I suppose that is like ``.set_precision(5)``? 

maybe raise here if its not a callable or string?